Documentation/ceph-cluster-crd.md

```diff
@@ -361,6 +361,71 @@ spec:
           memory: "4096Mi"
 ```
 
+### Custom Location Information On Node Level
+For each individual node a `location` can be configured. The provided information is fed directly into the CRUSH map of Ceph. More information on CRUSH maps can be found in the [ceph docs](http://docs.ceph.com/docs/master/rados/operations/crush-map/).
+
+**HINT** When setting this prior to `CephCluster` creation, these settings take immediate effect. However, applying this to an already deployed `CephCluster` requires to remove each node from the cluster first and then re-add it with new configuration to take effect. Do this node by node to keep your data safe! You can check the result with `ceph osd tree` from the [Rook Toolbox](ceph-toolbox.md) in your setup. The OSD tree should display your location hierarchy for the nodes you already re-added. 
```

Comment:
Just to confirm manually setting/moving the node's OSD(s) using `ceph` commands, didn't work for you?

Reply:
I tried using `ceph osd crush set osd.0 1.0 root=default datacenter=whichever` but that resulted in a failure indicating the osd wasn't in the correct state.

Reply:
Okay, weird.
@christianhuening the `location` has been correctly set in the ceph.conf after the Rook Ceph prepare job ran, right?

@travisn @liewegas any idea why changing the OSD location does not work in Rook's context here or behaving like this?

Reply:
@galexrt yes the ceph.conf of the OSDs did contain the location properly

Reply:
I'll go ahead and merge the docs PR now.

Though I'm hoping for a response from @travisn @liewegas to see why the `ceph osd crush` don't work.

Reply:
I was using Ceph 13.2.4 btw